(Doc+) link videos for allocation and ilm (#116880)
elastic/elasticsearch: docs/reference/troubleshooting/common-issues/red-yellow-cluster-status.asciidoc

In many cases, your cluster will recover to green health status automatically.
If the cluster doesn't automatically recover, then you must <<fix-red-yellow-cluster-status,manually address>>
the remaining problems so management and cleanup activities can proceed.

> See this https://www.youtube.com/watch?v=v2mbeSd1vTQ[monitoring Allocation health video]
> for an example troubleshooting walkthrough.

Reviewer: ```suggestion
See https://www.youtube.com/watch?v=v2mbeSd1vTQ[this video]
for a walkthrough of monitoring allocation health.
```

}
----

> See this https://www.youtube.com/watch?v=MiKKUdZvwnI[fixing "no allocations are allowed" video] for example troubleshooting walkthrough.

Reviewer: ```suggestion
See https://www.youtube.com/watch?v=MiKKUdZvwnI[this video] for walkthrough of troubleshooting "no allocations are allowed".
```

----
// TEST[s/^/PUT my-index\n/]
// TEST[catch:bad_request]

> See this https://www.youtube.com/watch?v=6OAg9IyXFO4[fixing "no_valid_shard_copy" video] for an example troubleshooting walkthrough.

Reviewer: ```suggestion
See https://www.youtube.com/watch?v=6OAg9IyXFO4[this video] for a walkthrough of troubleshooting `no_valid_shard_copy`.
```
